RELEASE/scripts/sl_ascend.ash

```diff
@@ -12544,6 +12555,14 @@ boolean L9_oilPeak()
 				cli_execute("make " + $item[Jar Of Oil]);
 			}
 			set_property("sl_oilpeak", "finished");
+
+			// Reset basline
+			set_property("sl_maximize_baseline", "");
```

Comment:
This property should only be touched by the user. I think you're looking for `resetMaximize()`